configParser.js

```diff
@@ -59,6 +59,17 @@ if (mongoOpts.ssl) {
   mongoOpts.sslCert = fs.readFileSync(mongoOpts.sslCert, 'utf8');
 }
 
+//warn user if their collection names are too large to help debug errors in configuration
+let collectionNameTooLarge = false;
+CONFIG.mongo.collections.forEach((collection) => {
+  if(collection.length > 110){
```

Comment:
Add a comment about the limit being 120 bytes and a link to this doc https://docs.mongodb.com/manual/reference/limits/#Restriction-on-Collection-Names

Reply:
Also you could do something like this instead to be more explicit
```
const COLL_NAME_MAX_LENGTH;
CONFIG.mongo.collections.forEach((collection) => {
  if(collection.length > COLL_NAME_MAX_LENGTH){
    console.log(`WARNING: The collection ${collection} is ${COLL_NAME_MAX_LENGTH - collection.length} characters too long.  Mongo does not allow names to be larger that 120 bytes.`)
  }
});```

Reply:
good suggestions.  Done